website/src/templates/models.js

```diff
@@ -134,6 +135,13 @@ function formatAccuracy(data, lang) {
         .filter(item => item)
 }
 
+function formatDownloadUrl(lang, name, version) {
+  const fullName = `${lang}_${name}-${version}`
+  const filename = `${fullName}-py3-none-any.whl`
+  const url = `https://github.com/explosion/spacy-models/releases/download/${fullName}/${filename}`
+  return <Link to={url} hideIcon>{filename}</Link>
```

Comment:
If it's called a URL I think it should look like a URL, or it should be called something else, like a "link".
```suggestion
  return <Link to={url} hideIcon>{url}</Link>
```

Reply:
I originally had the URL but changed it to the filename because I felt it was too long. We can use the URL here but I feel it's not helpful...

Reply:
If you have a better name please feel free to change it. I do feel that if we called it something like "Package link" it might not be clear that clicking on it would trigger a download.